<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QMainWindow" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>551</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit_ip">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_ip">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;URW Gothic L&quot;; border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>IP Address</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chat">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>280</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Android Emoji</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(158, 204, 143); text-color: &quot;white&quot;; border-radius: 15px; color: #333333;</string>
      </property>
      <property name="text">
       <string>Chat!</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;URW Gothic L&quot;; border-radius: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_port">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;URW Gothic L&quot;; border-radius: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_title">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>421</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 32pt &quot;VL Gothic&quot;; </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>C H A T</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_port">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_error">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">text-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_logs">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>330</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Carlito&quot;;</string>
      </property>
      <property name="text">
       <string>Save Logs</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_tag">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>341</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>The best linux chat client!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPlainTextEdit" name="plainTextEdit_msg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255); border-radius: 5px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_sendmsg">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>270</y>
        <width>95</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(158, 204, 143); text-color: &quot;white&quot;; border-radius: 8px; text-color: &quot;white&quot;;</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_messages">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>441</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 255, 255); border-radius: 5px;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="autoScrollMargin">
       <number>0</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>Messages</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
